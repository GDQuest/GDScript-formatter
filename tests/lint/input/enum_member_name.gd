#region Good

enum GoodElement {
	EARTH,
	WATER,
	AIR,
	FIRE,
}

#endregion

#region Bad

enum BadElement {
	Earth,
	water,
	cold_air,
	_fire,
}

#endregion