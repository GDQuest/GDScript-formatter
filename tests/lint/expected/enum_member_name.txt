tests/lint/input/enum_member_name.gd:15:enum-member-name:error: Enum element name 'Earth' should be in CONSTANT_CASE format
tests/lint/input/enum_member_name.gd:16:enum-member-name:error: Enum element name 'water' should be in CONSTANT_CASE format
tests/lint/input/enum_member_name.gd:17:enum-member-name:error: Enum element name 'cold_air' should be in CONSTANT_CASE format
tests/lint/input/enum_member_name.gd:18:enum-member-name:error: Enum element name '_fire' should be in CONSTANT_CASE format